src/text/Word2Vec.ts

```diff
@@ -0,0 +1,126 @@
+import * as tf from '@tensorflow/tfjs';
```

Comment:
See https://github.com/imgcook/datacook/tree/0ea4cab13f9972409b81713df2e0d0f8c4da25e2/src/text, shall we have a consistent to name our module here?

Reply:
ok